### ISSUE TITLE
Accessibility: provide ways for the Carousel auto rotation behavior to be stopped

### ISSUE THREAD

**Issue by sarah-cul** [projectblacklight/spotlight#3577]
### Issue Description - WCAG standard

https://www.w3.org/WAI/ARIA/apg/patterns/carousel/

If the carousel can automatically rotate, it also:

- Has a button for stopping and restarting rotation. This is particularly important for supporting assistive technologies operating in a mode that does not move either keyboard focus or the mouse.

- Stops rotating when keyboard focus enters the carousel. It does not restart unless the user explicitly requests it to do so.

- Stops rotating whenever the mouse is hovering over the carousel.
